.github/workflows/ci-pullrequest-main.yml

```diff
@@ -75,10 +80,64 @@ jobs:
       with:
         name: mutation-report
         path: src/StrykerOutput
+        
+  IntegrationTest:
+    runs-on: ubuntu-latest  
+    needs: BuildUnitTest
+
+    steps:
+    - uses: actions/checkout@v3
+    
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v2
+      with:
+        dotnet-version: 6.0.x
+        
+    - name: Checkout Test repo
+      uses: actions/checkout@v3
+      with:
+        repository: buerokratt/Tests
+        path: Tests
+        
+    - name: Checkout Mock Classifier repo
+      uses: actions/checkout@v3
+      with:
+        repository: buerokratt/Mock-Classifier
+        path: Mock-Classifier
+        
+    - name: Checkout DMR repo
+      uses: actions/checkout@v3
+      with:
+        repository: buerokratt/DMR
+        path: DMR
+        
+    - name: Checkout CentOps repo
+      uses: actions/checkout@v3
+      with:
+        repository: buerokratt/CentOps
+        path: CentOps
+      
+    - name: Create Docker network bridge
+      run: docker network create -d bridge my-network
+      
+    - name: Run Docker Compose Up to start containers
+      run: docker compose --file Tests/docker-compose.yml up --build --detach
+
+    - name: Restore dependencies for Test repo
+      run: dotnet restore Tests/src/Tests.IntegrationTests/*.csproj
+      
+    - name: Build Test repo
+      run: dotnet build Tests/src/Tests.IntegrationTests/*.csproj --no-restore
+       
+    - name: Test Test repo
+      run: dotnet test Tests/src/Tests.IntegrationTests/*.csproj --no-build
+      
+    - name: Run Docker Compose Down to stop containers
```

Comment:
Can we capture the logs from docker compose and output it here?

Reply:
Now capturing logs via https://github.com/marketplace/actions/collect-docker-logs. You can see this working on this pipeline run https://github.com/buerokratt/Mock-Bot/runs/7406695617?check_suite_focus=true